ext/opcache/jit/zend_jit_ir.c

```diff
@@ -14377,7 +14377,7 @@ static int zend_jit_fetch_obj(zend_jit_ctx         *jit,
 					ref = ir_CONST_ADDR(prop_info);
 				} else {
 					int prop_info_offset =
-						(((prop_info->offset - (sizeof(zend_object) - sizeof(zval))) / sizeof(zval)) * sizeof(void*));
+						(((prop_info->prototype->offset - (sizeof(zend_object) - sizeof(zval))) / sizeof(zval)) * sizeof(void*));
```

Comment:
Actually, this should only use `prototype` if it is not virtual, same as in `zend_build_properties_info_table()`. We can add a macro for this.